Unix/installbuilder/datafiles/AIX.data

```diff
@@ -41,6 +41,19 @@ RemoveOmiService() {
 %Preinstall_10
 #include OmiService_funcs
 RemoveOmiService > /dev/null 2> /dev/null
+egrep "^omi:" /etc/group 2>&1 >/dev/null
+if [ $? -ne 0 ]; then
+    echo "Creating omi group ..."
+    mkgroup -a omi
```

Comment:
For much of this, there's a question of if we should use fully-qualified paths to the appropriate tools. If you don't do this, and if the environment is inherited by the installers, then unintended consequences may occur.

In the past, KRBash forbid this sort of thing. He's not here now, though, so we should discuss it as a group and reach consensus. We should probably bring it up at the end of scrum tomorrow (Thursday).